src/ci/docker/scripts/cross-apt-packages.sh

```diff
@@ -22,6 +22,7 @@ apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install
   patch \
   pkg-config \
   python3 \
+  rsync \
```

Comment:
`rsync` used somewhere? Can't see any matches here.

Reply:
It is used by `crosstool-ng` git version.